### ISSUE TITLE
Extra "true" in `printfmt` output

### ISSUE THREAD

**Issue by chris-b1** [JuliaString/Format.jl#81]
E.g. 
```julia
julia> printfmt("{:d}", 1000)
1000true
```

Doesn't occur in `printfmtln` or `pyfmt`

Version info
```julia
(jl_atRSS9) pkg> st
Status `/tmp/jl_atRSS9/Project.toml`
  [1fa38f19] Format v1.3.6

julia> versioninfo()
Julia Version 1.10.0
Commit 3120989f39b (2023-12-25 18:01 UTC)
Build Info:
  Official https://julialang.org/ release
Platform Info:
  OS: Linux (x86_64-linux-gnu)
  CPU: 16 × 12th Gen Intel(R) Core(TM) i7-1270P
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-15.0.7 (ORCJIT, alderlake)
  Threads: 1 on 16 virtual cores
```

**Comment by ScottPJones** [comment]
The "extra" `true` is not being actually being output by the print (as you'd see if it happened in a function), it just is an artifact of the REPL displaying whatever a function returns (unless it is `nothing`)
Still, it's an annoyance, and I'll fix this!

**Comment by ScottPJones** [comment]
Fixed by #83

**Comment by chris-b1** [comment]
Great, thanks!
